docs/src/test/java/jdocs/stream/operators/source/Iterate.java

```diff
@@ -0,0 +1,13 @@
+package jdocs.stream.operators.source;
+
+import org.apache.pekko.NotUsed;
+import org.apache.pekko.stream.javadsl.Source;
+
+interface Iterate {
+
+  // #countTo
+  static Source<Long, NotUsed> countTo(long n) {
+    return Source.iterate(1L, i -> i + 1).take(n);
```

Comment:
For comparison, it may be more friendly to use the same API as ScalaDsl.

```suggestion
    return Source.iterate(1L, i -> true, i -> i + 1).take(n);
```